client/view/Form/form.js

```diff
@@ -29,17 +35,53 @@ export function initForm() {
         once: true
       }
     )
-    updateStatsView(query)
+    updateStatsView(query, region)
   })
+
   textarea.addEventListener('keypress', e => {
     if (e.keyCode === 13 && !e.shiftKey) {
-      e.preventDefault()
-      form.dispatchEvent(new Event('submit'))
+      submitForm()
     }
   })
+
+  regionSelect.addEventListener('change', submitForm)
+}
+
+function submitForm() {
+  form.dispatchEvent(new Event('submit', { cancelable: true }))
```

Comment:
@ariarzer I fixed this bug :)

https://github.com/browserslist/browserl.ist/issues/310#issuecomment-1205180113